src/api/object_api.js

```diff
@@ -535,6 +535,33 @@ module.exports = {
             }
         },
 
+        read_upload_md: {
+            method: 'GET',
+            params: {
+                type: 'object',
+                required: [
+                    'bucket',
+                    'key',
+                ],
+                properties: {
+                    bucket: { $ref: 'common_api#/definitions/bucket_name' },
+                    key: { type: 'string' },
+                }
+            },
+            reply: {
+                type: 'object',
+                properties: {
+                    obj_id: { objectid: true },
+                    key: { type: 'string' },
+                    xattr: { $ref: '#/definitions/xattr' }
+                }
+            },
+            auth: {
+                system: ['admin', 'user'],
+                anonymous: true,
```

Comment:
why is that anonymous? really needed? CC @jeniawhite

Reply:
deleted from api